src/sections/PairSection.tsx

```diff
@@ -120,6 +158,39 @@ export default function PairSection() {
   );
 }
 
+const SkeletonPariPlaceholder = () => {
```

Comment:
For the code convention, Please move utility components, constant variables, and utility functions ahead of the main component.

Reply:
done